src/Web/Microsoft.NET.Sdk.Web.Targets/Sdk.props

```diff
@@ -10,6 +10,15 @@ Copyright (c) .NET Foundation. All rights reserved.
 ***********************************************************************************************
 -->
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--
+      Indicate to other targets that Microsoft.NET.Sdk.Web is being used.
+      This must be set here (as early as possible, before Microsoft.Common.props)
+      so that everything that follows can depend on it.
+      In particular, Directory.Build.props and nuget package props need to be able
+      to use this flag and they are imported by Microsoft.Common.props.
+    -->
+    <UsingMicrosoftNETSdkWeb>true</UsingMicrosoftNETSdkWeb>
```

Comment:
We'd like to set [some items](https://github.com/aspnet/websdk/blob/master/src/ProjectSystem/Microsoft.NET.Sdk.Web.ProjectSystem.Targets/netstandard1.0/Microsoft.NET.Sdk.Web.ProjectSystem.targets#L47-L49) in the Razor.Sdk if we know Web.Sdk isn't being referenced. I think it's safe to set duplicate capabilities, but this might help in the future for us to determine if the Razor sdk is being directly referenced versus referenced via Web.Sdk